bindings/haskell/if-proto-scip-hs.cabal

```diff
@@ -0,0 +1,21 @@
+cabal-version:      3.6
+
+name:               if-proto-scip-hs
+version:            0.1.0.0
+synopsis:           Haskell bindings for SCIP Code Intelligence Protocol
+license:            Apache-2.0
+license-file:       LICENSE
+author:             SCIP Maintainers
+maintainer:         code-intel@sourcegraph.com
+homepage:           https://github.com/sourcegraph/scip
+category:           Language
+build-type:         Simple
+
+library
+    ghc-options: -Wall
+    exposed-modules:
+        Proto.Scip,
+        Proto.Scip_Fields
+    hs-source-dirs:   src
+    default-language: Haskell2010
+    build-depends: proto-lens-runtime, base
```

Comment:
We should specify some version ranges here. ([examples](https://cabal.readthedocs.io/en/3.4/cabal-package.html))

We can probably add a freeze file (analogous to lock files elsewhere) later.